.circleci/config.yml

```diff
@@ -11,10 +13,12 @@ jobs:
   run-specs-with-postgres:
     executor: solidusio_extensions/postgres
     steps:
+      - browser-tools/install-browser-tools
```

Comment:
We'll have to add this step to the CircleCI configurations of all extensions which have integration tests.